components/suggestion/switch_channel_provider.jsx

```diff
@@ -142,15 +142,17 @@ class SwitchChannelSuggestion extends Suggestion {
                 onClick={this.handleClick}
                 onMouseMove={this.handleMouseMove}
                 className={className}
+                role='list'
                 ref={(node) => {
                     this.node = node;
                 }}
                 id={`switchChannel_${channel.name}`}
                 data-testid={channel.name}
+                aria-label={channel.name}
```

Comment:
Shouldn't this be the display name?

Reply:
same here

Reply:
Oh, they don't have their `displayName` set because that's injected into the "My Channels" by those selectors I hate 😛. Maybe we could do `displayName || channel.name` here instead?